packages/npm/@amazeelabs/bridge-gatsby/src/index.tsx

```diff
@@ -14,12 +14,21 @@ export const Link: LinkType &
 
 export const useLocation: useLocationType = () => {
   const location = gatsbyUseLocation();
+  const [updatedSearch, setUpdatedSearch] = React.useState('');
```

Comment:
Is the search enough? What about the query fragment, can we test if that works correctly?
Otherwise we maybe should store the whole url object in the state?